xml/security-barbican.xml

```diff
@@ -197,35 +164,43 @@
     </para>
     <warning>
      <para>
-      Do not change your master key after deploying it to Barbican.
+      Do not change your master key after deploying it to &secret_store;.
      </para>
     </warning>
    </listitem>
    <listitem>
     <para>
      If you decide to make configuration changes to your clean install of
-     &kw-hos-phrase;, you will need to redeploy the Barbican service. For
+     &kw-hos-phrase;, you will need to redeploy the &secret_store; service. For
      details on available customization options, please see
      <xref linkend="barbicanAdmin"/>.
     </para>
    </listitem>
   </itemizedlist>
  </section>
- <section>
-  <title>Error Received if Root Logical Volume is Too Small</title>
+ <section xml:id="sec.barbican.kmip">
+  <title>Master Key Configuration</title>
   <para>
-   Barbican currently supports databases and KMIP as its secret store back-ends
-   In &ostack-bare; upstream additional back-ends are available, such as the
-   PKCS11 and dogtag plug-ins, but they are not tested or supported by
+   &secret_store; currently supports databases and KMIP as its secret store
+   back-ends In &ostack-bare; upstream additional back-ends are available, such
```

Comment:
add a full stop after `back-ends.`, add a comma between `upstream, additional` (this commit is not at fault here, it just makes the issues more obvious.)